src/gui/qgsadvanceddigitizingdockwidget.cpp

```diff
@@ -141,20 +144,56 @@ QgsAdvancedDigitizingDockWidget::QgsAdvancedDigitizingDockWidget( QgsMapCanvas *
   mLockYButton->setToolTip( "<b>" + tr( "Lock y coordinate" ) + "</b><br>(" + tr( "press Ctrl + y for quick access" ) + ")" );
   mRepeatingLockYButton->setToolTip( "<b>" + tr( "Continuously lock y coordinate" ) + "</b>" );
 
+  // Create the slots/signals
+  connect( mXLineEdit, &QLineEdit::textChanged, this, &QgsAdvancedDigitizingDockWidget::valueXChanged );
+  connect( mYLineEdit, &QLineEdit::textChanged, this, &QgsAdvancedDigitizingDockWidget::valueYChanged );
+  connect( mDistanceLineEdit, &QLineEdit::textChanged, this, &QgsAdvancedDigitizingDockWidget::valueDistanceChanged );
+  connect( mAngleLineEdit, &QLineEdit::textChanged, this, &QgsAdvancedDigitizingDockWidget::valueAngleChanged );
+
+
+  // Create the floater
+  mFloater = new QgsAdvancedDigitizingFloater( canvas, this );
+  mToggleFloaterAction->setChecked( QgsSettings().value( QStringLiteral( "/Cad/Floater" ), false ).toBool() );
+  toggleFloater( QgsSettings().value( QStringLiteral( "/Cad/Floater" ), false ).toBool() );
 
   updateCapacity( true );
   connect( QgsProject::instance(), &QgsProject::snappingConfigChanged, this, [ = ] { updateCapacity( true ); } );
 
   disable();
 }
 
+
 void QgsAdvancedDigitizingDockWidget::hideEvent( QHideEvent * )
 {
   // disable CAD but do not unset map event filter
   // so it will be reactivated whenever the map tool is show again
-  setCadEnabled( false );
+  // TODO : remove this (we don't want to disable CAD events, even when the dockerwidget is hidden)
```

Comment:
What's this TODO exactly?

Reply:
Removed the whole hideEvent function, since we don't want anymore to disable cad when the dockwidget is closed